src/main/java/duke/command/DukeActions.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.util.DukeConfig;
+import duke.util.DukeDB;
+import duke.util.DukeException;
+import duke.util.DukeTaskList;
+import duke.util.Ui;
+
+import java.util.Map;
+
+
+/**
+ * The DukeActions interfaces encapsulates the run method for the any command.
+ */
+public interface DukeActions {
```

Comment:
Is there a reason you use an interface instead of an abstract class? Since the example given on the iP website was to have an abstract class Command.